CHANGELOG.md

```diff
@@ -4,9 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 0.1.3 - 20178-02-08
+## 0.2.0 - 2018-03-14
+Interaction release
+
+### Added
+* New `clearUploads()` method to remove current ulpoads.
+* New `reset()` method reset widget mannualy.
+
+### Changed
+* Now component is interactive, widget accept changes of input parameters.
+* Updated README with info about install, usage, etc.
+* Updated Demo projects with examples of usage
```

Comment:
* The component is now interactive: your widget `reacts to | accepts` the changes in input parameters.

```
Or, something like, "...widget changes its look and behavior based on the changes in input parameters."
```
* Updated the README: install, usage, etc.
* Updated the `[Demo projects](<link>)` with usage examples